vsphere/resource_vsphere_resource_pool_test.go

```diff
@@ -0,0 +1,222 @@
+package vsphere
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-vsphere/vsphere/internal/helper/viapi"
+)
+
+func TestAccResourceVSphereResourcePool_basic(t *testing.T) {
```

Comment:
There should be a check here that checks that the expected configuration settings are present in the resource pool. This would be a function that would check the pertinent (if not all) resource pool settings, possibly in a fashion that it can be re-used in future tests (such as the update test).